lib/auth.js

```diff
@@ -231,6 +234,47 @@ module.exports = new Model({
     }.bind(this));
   },
 
+  signInWithOAuthRedirect: function (args, window) {
```

Comment:
It does, but it then makes it harder to test - I couldn't find a decent way of mocking the window object that didn't require using a global.